obal/data/modules/find_tags.py

```diff
@@ -0,0 +1,31 @@
+"""
+Find all defined Koji tags in inventory
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+def main():
+    """
+    Verify a tag
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            packages=dict(type='dict', required=True)
+        )
+    )
+
+    packages = module.params['packages']
+
+    tags = []
+
+    for (package, attributes) in packages.items():
+        if 'koji_tags' in attributes:
+            for tag in attributes['koji_tags'].keys():
```

Comment:
```suggestion
       for tag in attributes.get('koji_tags', {}).keys():
```

But isn't `koji_tags` a list, not a dict?

Reply:
It was, but a dictionary makes more sense -- https://github.com/theforeman/obal/pull/241